vite.config.ts

```diff
@@ -12,20 +20,10 @@ export default defineConfig({
       fileName: (format) => `react-section-dividers.${format}.js`,
     },
     rollupOptions: {
-      external: [
-        'react',
-        'react-dom',
-        '@emotion/cache',
-        '@emotion/hash',
-        '@emotion/is-prop-valid',
-        '@emotion/memoize',
-        '@emotion/react',
-        '@emotion/serialize',
-        '@emotion/styled',
-        '@emotion/styled/base',
-        '@emotion/unitless',
-        '@emotion/utils',
-      ],
+      external: makeExternalPredicate([
```

Comment:
The problem was that `external` is only handling **exact** matches by default and you were also importing from `@emotion/react/jsx-runtime` and that dragged a copy of Emotion to the bundle.